manifests/klusterlet/management/klusterlet-agent-deployment.yaml

```diff
@@ -115,3 +124,8 @@ spec:
       - name: hub-kubeconfig
         emptyDir:
           medium: Memory
+      {{if eq .InstallMode "SingletonHosted"}}
+      - name: spoke-kubeconfig-secret
+        secret:
+          secretName: {{ .ExternalManagedKubeConfigWorkSecret }}
```

Comment:
for singletonHosted mode, will it still generate two(external registration/work) secrets? why use the work secret here?

Reply:
yeah, it is a bit confusing. Updated and add some comment.